pom.xml

```diff
@@ -104,6 +110,13 @@
             <version>${jenkins.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-all</artifactId>
+            <version>1.3</version>
+            <scope>test</scope>
+        </dependency>
+        

```

Comment:
Do you need both hamcrest-all and hamcrest-core dependencies? Doesn't hamcrest-all pull in hamcrest-core?